dotcom-rendering/src/web/layouts/LiveLayout.tsx

```diff
@@ -961,12 +938,12 @@ export const LiveLayout = ({ CAPI, NAV, format, palette }: Props) => {
 														<Pagination
 															currentPage={
 																CAPI.pagination
-																	?.currentPage ||
+																	?.currentPage ??
```

Comment:
I don’t know about that… I couldn’t figure out the code here, really. Can the page be 0-indexed? What happens in this case? This is especially relevant for `number` types which can be `0`. 

Maybe the code should be left as-is, but I actually find it harder to reason about as it is.

Reply:
It is without doubt true that using ternaries for conditional rendering is never going to be the clearest code no matter what pattern you use! 🙃 

It's also true that the `0` business is a classic react gotcha. We add the additional check for `thing === 0` for this but it's easy to forget this.

I guess we want the DCR code to be idiomatic. It should as closely resemble the sort of code you see in examples or in tutorials so that when new developers, esp. more junior ones or people new to react, open a file they see as much familiarity as possible.

I think I'm just very keen that we try to keep the number of patterns and things new people need to learn to a minimum. We will all of us leave this team at some point and new people will join with their own backgrounds and experiences. The common threads between us are these idiomatic patterns so even if another approach is objectively 'better' - and I suspect you're right in this - maybe the bigger win of approachable code supersedes this?

Reply:
Ok, let’s extract them, then?

```ts

const currentPage: number = CAPI.pagination?.currentPage || 1; // minimum of 1
const totalPages: number = CAPI.pagination?.totalPages || 1; // minimum of 1

```

Reply:
👍 

(❤️ for the little explainer comment)

Reply:
I went for a slightly different approach of setting a default pagination. Everything still works the same.